fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/common/CheckPointAlgorithm.java

```diff
@@ -95,16 +95,18 @@ public boolean isReadyToCheckpoint() {
                     return (chunkData.getBufferStream().size() >= cosMaxFileSize
                             || chunkData.getPageNum() > chunkData.getLastPageNum());
                 } else {
-                    try {
-                        numofPagePerCosObject = Integer.parseInt(pagesPerCosObject);
-                        logger.info("isReadyToCheckpoint: " + numofPagePerCosObject + " pages per COS object!");
-                    } catch (Exception e) {
-                        numofPagePerCosObject = Constants.DEFAULT_NUMOFPAGES_EACH_COS_OBJECT;
-                        logger.warning("isReadyToCheckpoint: Set number of pages per COS object to default("
-                                + Constants.DEFAULT_NUMOFPAGES_EACH_COS_OBJECT + ").");
+                    if (pagesPerCosObject != null) {
+                        try {
+                            numofPagePerCosObject = Integer.parseInt(pagesPerCosObject);
+                            logger.info("isReadyToCheckpoint: " + numofPagePerCosObject + " pages per COS object!");
```

Comment:
```suggestion
                            logger.fine("isReadyToCheckpoint: [" + numofPagePerCosObject + "] pages per COS object!");
```
should this be set to fine?

Reply:
make sense, will change